ecc/enclave/cgo_logging.go

```diff
@@ -0,0 +1,17 @@
+/*
+   Copyright 2019 Intel Corporation
+   Copyright IBM Corp. All Rights Reserved.
+
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package enclave
+
+// extern void golog(char*);
+//
+// int golog_cgo_wrapper(const char* str)
+// {
+//      golog((char*)str);
```

Comment:
should we also move `golog()` from `enclave_stub.go` also to this file?

Reply:
this is is also the change to clean-up the unused `_logger` in `ecc/enclave/enclave_stub.go`.

Reply:
and we even `var logger = flogging.MustGetLogger("ecc_enclave")` go be moved to this logging file. so we have everything related to logging in a single place

Reply:
this file could also be just called `logging.go` I think

Reply:
@mbrandenburger if you move the logger here then you loose the context specific logger we currently have.  As mentioned above ("This is a duplicate version ...") i think you might be able to get the best of both worlds by passing the logger via some closure.

Reply:
> then you loose the context specific logger we currently have.
admittedly, for any particular program, there is only one context/logger, so it really doesn't matter if for enclave logging it says tlcc_enclave or ecc_enclave or alike as the context is clear. So just having to logger object in internal/logging.go would be considerably easier than what i suggest and perfectly sufficient :-)

Reply:
not sure if I really understand this. But moving `var logger = flogging.MustGetLogger("ecc_enclave")` into `cgo_logging.go` will retain the contexts specific logger as it is still in the same package just in a different file.